### PR TITLE
Update CVE-2018-6660.json

### PR DESCRIPTION
**Pull request by htoomey**
SB10228 ePO Directory Traversal vulnerability

### CODE REVIEW

File: 2018/6xxx/CVE-2018-6660.json [hunk 1]
```diff
@@ -1,18 +1,83 @@
 {
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6660.json]: JSON: u'version_value' is a required property

**Comment by cve-team** [comment]
Auto-validation for this pull request has detected issues:
JSON: Please see review comments for errors.

**Comment by htoomey** [comment]
Please reject this pull request.  Use the new one submitted with correct JSON data.

**Comment by cve-team** [comment]
Marking as withdrawn.

**Pull request closed without merging**